src/test/munit/sftp-write-knowhosts-test-case.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:munit="http://www.mulesoft.org/schema/mule/munit"
+      xmlns:munit-tools="http://www.mulesoft.org/schema/mule/munit-tools"
+      xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:sftp="http://www.mulesoft.org/schema/mule/sftp"
+      xmlns:java="http://www.mulesoft.org/schema/mule/java"
+      xsi:schemaLocation="
+		http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+		http://www.mulesoft.org/schema/mule/munit http://www.mulesoft.org/schema/mule/munit/current/mule-munit.xsd
+		http://www.mulesoft.org/schema/mule/munit-tools  http://www.mulesoft.org/schema/mule/munit-tools/current/mule-munit-tools.xsd
+        http://www.mulesoft.org/schema/mule/sftp http://www.mulesoft.org/schema/mule/sftp/current/mule-sftp.xsd
+        http://www.mulesoft.org/schema/mule/java http://www.mulesoft.org/schema/mule/java/current/mule-java.xsd">
+
+    <munit:config name="sftp-write-knowhosts-test-case.xml"/>
+
+    <configuration-properties file="automation-credentials.properties"/>
+
+    <sftp:config name="config-known-host">
+        <sftp:connection username="${sftp.user}" password="${sftp.password}" host="44.217.240.197" port="${sftp.port}"
+                         knownHostsFile="know_hosts.properties" workingDir="/upload">
+        </sftp:connection>
+    </sftp:config>
+
+    <sftp:config name="config-known-host-error">
+        <sftp:connection username="${sftp.user}" password="${sftp.password}" host="44.217.240.197" port="${sftp.port}"
+                         knownHostsFile="know_hosts_error.properties" workingDir="/upload">
+        </sftp:connection>
+    </sftp:config>
+
+    <munit:test name="sftp-write-file-know-host">
+        <munit:behavior>
+            <set-variable variableName="fileName" value="write.txt"/>
+            <set-variable variableName="mode" value="OVERWRITE"/>
+            <set-variable variableName="createParent" value="true"/>
```

Comment:
I comprehend that the fileName has been assigned to a variable due to its subsequent usage. However, the other two variables can be directly employed within the code, wouldn't you agree?